apps/prairielearn/src/api/trpc/routers/course-files/batchDeleteQuestions.ts

```diff
@@ -38,18 +33,25 @@ export const deleteQuestion = privateProcedure
       authn_user_id: opts.input.authn_user_id,
     });
 
-    const question = await selectQuestionById(opts.input.question_id);
+    const questions = await queryRows(
```

Comment:
also does a double-check to avoid deleting questions out-of-course. should we add a length check to throw an error on non-existent or out-of-course question?